CMakeLists.txt

```diff
@@ -324,7 +324,3 @@ message(STATUS "Linter: clang-format            = ${OPENASSETIO_ENABLE_CLANG_FOR
 
 add_subdirectory(src)
 add_subdirectory(examples)
-
-if (OPENASSETIO_ENABLE_TESTS)
-    add_subdirectory(tests)
-endif ()
```

Comment:
[Question] Is this the right commit for this? This removes the entire test directory from the top level CMakeLists, which isn't just python to my knowledge. I guess because it's the last commit, it's remained in to keep each commit functional up until now?

Reply:
Yeah exactly, the last vestige of the old `tests` top-level directory is gone with the move of Python tests (C++ and C tests were moved in previous commits), so these lines can finally be removed at this point.